src/Log/Engine/SyslogLog.php

```diff
@@ -94,7 +94,8 @@ public function __construct(array $config = [])
         if (isset($config['format'])) {
             deprecationWarning(
                 '`format` option is now deprecated in favor of custom formatters. ' .
-                'Switching to `LegacySyslogFormatter`.'
+                'Switching to `LegacySyslogFormatter`.',
+                0
```

Comment:
Won't this result in the deprecation file reference now pointing to the constructor of SyslogLog?

Reply:
Not sure about this one. I don't know if the parent stack frame is relevant or if it's just buried deep in a config factory.

Reply:
True, it is likely pointing to a different unhelpful place.